client/src/locales/en.json

```diff
@@ -10,7 +10,8 @@
   "app.toolbar.go-to-dashboard": "Go back to dashboard",
   "app.toolbar.notification-tooltip": "See notifications",
   "common.archived-message.restoring": "Restoring '{name}' {item}...",
-  "common.archived-message.was-archived": "The '{name}' {item} was archived.",
+  "common.archived-message.was-archived": "The '{name}' {item} was archived. ",
+  "common.archived-message.was-archived-member": "The '{name}' {item} was archived. To get access to the data please contact the owner to restore '{name}' {item}.",
```

Comment:
`... restore the '{name}' ...`

Reply:
Please split out the common prefix, then you can use separate messages and display the additional message if needed. By combining and duplicating locales, the translation costs increase.